### ISSUE TITLE
Customise with learning rate

### ISSUE THREAD

**Issue by anushka23g** [anushka23g/Learning-Made-Easy#2]
Add a functionality, which keeps a check for each time you assign a schedule, about what a person thinks about it. Take feedback, understand their learning rate and adapt accordingly.

**Comment by draveshjaluka** [comment]
can you elaborate about this issue

**Comment by anushka23g** [comment]
yes, basically what I'm planning to do is to store a variable as learning rate in a file and keep on updating it inside the file based on the user feedback function, this also needs some minor changes in the user feedback func. and then we'll use that variable to make the code efficient. 
Not exactly but something like the learning rate of neural networks

**Comment by draveshjaluka** [comment]
This is kind of machine learning

**Comment by draveshjaluka** [comment]
would love to work on it

**Comment by anushka23g** [comment]
yes, not exactly but thinking of something like that.
Assigned to @draveshjaluka

**Comment by draveshjaluka** [comment]
Thanks

**Comment by draveshjaluka** [comment]
Would also try to bring some display change in terminal only to make it look good

**Comment by anushka23g** [comment]
Please elaborate

**Comment by draveshjaluka** [comment]
Like the basic terminal display is not very well aligned would align it
